cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -218,8 +218,8 @@ public void execute() {
             targetPath = sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid Ballerina source path, it should either be a module name in a Ballerina project or a " +
-                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension.",
+                    "invalid Ballerina source path, it should either be a module name in a Ballerina project, a " +
```

Comment:
do we need the "in a Ballerina project" part?
```suggestion
                    "invalid Ballerina source path, it should either be a module name, a " +
```

Reply:
This error message seems to be consistent with all other commands. IMO stating 'in a Ballerina project' would be good for a newcomer who is experimenting.

Reply:
Shall we rephrase this as follows then 
```suggestion
                    "invalid Ballerina source path, it should either be a name of a module in a Ballerina project, a " +
```

Reply:
+1 , i'll make that change across all the commands.

Reply:
+1

Reply:
Fixed